scheduler/pkg/util/constants.go

```diff
@@ -21,5 +21,6 @@ import "time"
 const (
 	GrpcRetryBackoffMillisecs         = 100
 	GrpcRetryMaxCount                 = 5 // around 3.2s in total wait duration
+	GrpcMaxMsgSizeBytes               = 100 * 1024 * 1024
```

Comment:
is this aligning with max kafka payload size? as we are using it in the pipeline gateway.

Reply:
That's currently 1Gb so can increase by 10 here